tools/SetupFlow/DevHome.SetupFlow/Models/RepositoryProvider.cs

```diff
@@ -245,10 +245,13 @@ public RepositorySearchInformation SearchForRepositories(IDeveloperId developerI
                 else
                 {
                     _log.Error(result.Result.ExtendedError, $"Could not get repositories.  Message: {result.Result.DisplayMessage}");
+                    TelemetryFactory.Get<ITelemetry>().Log("RepoTool_SearchForRepos_Event", LogLevel.Critical, new GetReposEvent(result.Result.ExtendedError, _repositoryProvider.DisplayName, developerId));

```

Comment:
I agree on giving extensions localization independent identifiers.  I do not know how to go about that.  Also agree that this is a small issue because DevHome has 2 Repository Providers.  I'll think on this.